### PR TITLE
[WIP] Removing a feature

### PR DESCRIPTION
**Pull request by richardliaw**
Allowing better performance wrt opponent.

### CODE REVIEW

**Comment by richardliaw** [comment]
@dcrankshaw @pcmoritz @robertnishihara

File: rl_and_pong/pong_model_container.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def f():
         self.agent.restore(path)
         # Run test prediction to load the model
         print("Predicted {} in constructor".format(
-            self.agent.compute_action(np.random.random(8))))
+            self.agent.compute_action(np.random.random(7))))
```
Comment (dcrankshaw) [inline rl_and_pong/pong_model_container.py]: You can just comment out this whole print statement. The version I pushed to Dockerhub does that (see #36).

**Comment by dcrankshaw** [comment]
The code looks fine on my end. Does this help?

**Comment by richardliaw** [comment]
Seems to help a lot in training.

**Comment by dcrankshaw** [comment]
Testing now

**Comment by richardliaw** [comment]
One of the patches for Ray didn't get through, so there's a bit of a performance difference atm

**Comment by richardliaw** [comment]
@robertnishihara

**Pull request merged**